docs/source/plugins/plugins.rst

```diff
@@ -220,5 +220,4 @@ Available configuration options
 .. content-tabs:: left-col
 
     Your favorite monitoring is currently not supported?
-    Please, have a look at
-    :ref:`how to contribute.<ref_contribute_plugin>`
+    Please, have a look at how to contribute.
```

Comment:
Any reason for removing the link? Is this an artefact from another PR?

Reply:
The destination of the link does exists anymore. `sphinx` was throwing errors.